pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -778,7 +778,6 @@ func registerTPCC(r registry.Registry) {
 				Duration:                      10 * time.Minute,
 				ExtraRunArgs:                  "--wait=false --tolerate-errors --workers=200 --literal-implementation",
 				SetupType:                     usingImport,
-				ExpensiveChecks:               true, // Run expensive checks here to catch any issues with the literal implementation
```

Comment:
I intentionally had these enabled at first so that we could catch any bugs that I may have introduced with literal mode. The test has been running cleanly for a few weeks now, so I think we're good to disable the checks.